src/dotnet/14.1/Imports/Microsoft.Common.props/ImportBefore/Microsoft.NuGet.ImportBefore.props

```diff
@@ -0,0 +1,17 @@
+<!--
+***********************************************************************************************
+Microsoft.NuGet.ImportBefore.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <NuGetProps Condition="'$(NuGetProps)'==''">$(MSBuildExtensionsPath)\Microsoft.NuGet.props</NuGetProps>
```

Comment:
NuGetProps is getting set here, if it isn't already set.

MSBuildExtensionsPath is a property set by MSBuild itself.  It resolves to the same directory as `msbuild.exe` is running from, which is our `sdk\<version>` directory - which is where we are installing the Microsoft.NuGet.props file to.

Keep in mind, this file will only be loaded when using the CLI.  When using VS, or some other MSBuild installation, this ImportBefore.props file won't be loaded, since it only exists in the CLI directory.